src/ripple/app/misc/Validations.cpp

```diff
@@ -206,12 +206,12 @@ class ValidationsImp : public Validations
 
         auto const now = app_.timeKeeper().closeTime();
         auto const signTime = val->getSignTime();
-
+        auto const p = ConsensusParms{};
```

Comment:
I'm not sure how `p` will look in the replacement, but here it could be made `constexpr` instead of `const`, assuming VS will swallow that.  Doing so will ensure these constants are compile-time initialized, and will therefore be a little more efficient at run time.